uber/templates/registration/merch.html

```diff
@@ -3,127 +3,129 @@
 {% block content %}
 
 <script type="text/javascript">
```

Comment:
Since I was going through this anyway, I changed double quotes to single quotes and fixed some indentation.

The only "real" fix here is that we're now quoting the id since it's a uuid now and not an integer.